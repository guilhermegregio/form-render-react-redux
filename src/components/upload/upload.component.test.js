import React from 'react';
import ReactDOM from 'react-dom';

import Upload from './upload.component';

it('default render upload component', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Upload id="upload" name="upload" className="upload" />, div);

    expect(div).toMatchSnapshot();
});
