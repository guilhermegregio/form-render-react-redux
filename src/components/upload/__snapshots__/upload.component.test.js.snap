// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render upload component 1`] = `
<div>
  <div
    data-reactroot=""
    class="upload-component group upload"
  >
    <input
      type="file"
      id="upload"
      name="upload"
    />
  </div>
</div>
`;
